src/components/navigation-bar/navigation-bar.tsx

```diff
@@ -0,0 +1,59 @@
+import AddOutlinedIcon from '@mui/icons-material/AddOutlined';
+import { Button, List, ListItem, ListItemButton, ListItemIcon, ListItemText } from '@mui/material';
+import React, { ReactElement } from 'react';
+
+import {
+  AllInboxOutlinedIconNavBar,
+  DateRangeOutlinedIconNavBar,
+  PriorityHighOutlinedIconNavBar,
+  TodayOutlinedIconNavBar,
+  BoxNavBar,
+} from './navigation-bar.styles';
+
+const NavigationBar = (): ReactElement => (
+  <BoxNavBar>
+    <nav aria-label='main mailbox folders'>
```

Comment:
what is the point of nav if you use BoxNavBar upper?

Reply:
BoxNavBar is just a styles div